server/config/topic_page/poverty/India.textproto

```diff
@@ -0,0 +1,373 @@
+    }
+    left_tiles {
+      type: RANKING
+      ranking_metadata {
+        show_highest: true
+        show_lowest: true
+      }
+    }
+    right_tiles: {
+      type: LINE
+      title: "Poverty rate in ${place}"
+    }
+  }
+
+  blocks {
+    title: "Urban vs. Rural"
+    description: "Percentage of those below poverty who live in urban vs. rural areas"
+    stat_var_metadata {
+      stat_var: "Count_Person_Urban_BelowPovertyLevelInThePast12Months"
+      denom: "Count_Person_BelowPovertyLevelInThePast12Months"
+      unit: "%"
+      scaling: 100
+      name: "Share of poor in urban areas"
+    }
+    stat_var_metadata {
+      stat_var: "Count_Person_Rural_BelowPovertyLevelInThePast12Months"
+      denom: "Count_Person_BelowPovertyLevelInThePast12Months"
+      unit: "%"
+      scaling: 100
+      name: "Share of poor in rural areas"
+    }
+    left_tiles: {
+      title: "Urban vs. Rural among those below poverty in ${place}"
+      type: LINE
+    }
+    right_tiles {
+      type: RANKING
+      ranking_metadata {
+        show_highest: true
+      }
+    }
+  }
+
+  blocks {
+    title: "Literacy and poverty"
+    stat_var_metadata {
+      stat_var: "Count_Person_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person"
+      name: "Poverty rate"
+      unit: "%"
+    }
+    stat_var_metadata {
+      stat_var: "Count_Person_Illiterate"
+      denom: "Count_Person"
+      name: "Illiteracy rate"
+      unit: "%"
+    }
+    left_tiles {
+      title: "Poverty rate vs. Illiteracy rate"
+      type: SCATTER
+    }
+    right_tiles {
+      title: "Poverty rate vs. Illiteracy rate"
+      type: BIVARIATE
+    }
+  }
+
+  blocks {
+    title: "Temperature rise and poverty"
+    stat_var_metadata {
+      stat_var: "DifferenceRelativeToBaseDate2015_Max_Temperature_SSP585"
+      unit: "C"
+      name: "Max temp relative to 2015, RCP 8.5, SSP 5"
+    }
+    stat_var_metadata {
+      stat_var: "Count_Person_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person"
+      unit: "%"
+      name: "Poverty rate"
+    }
+    left_tiles {
+      title: "Max temp relative to 2015 (RCP 8.5, SSP 5) vs. Poverty rate"
+      type: SCATTER
+    }
+    right_tiles {
+      title: "Max temp relative to 2015 (RCP 8.5, SSP 5) vs. Poverty rate"
+      type: BIVARIATE
+    }
+  }
+}
+
+categories {
+  title: "Education and Poverty"
+
+  blocks {
+    title: "Schools per capita"
+    stat_var_metadata {
+      stat_var: "Count_School"
+      denom: "Count_Person"
+      name: "Schools per capita"
+    }
+    stat_var_metadata {
+      stat_var: "Count_Person_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person"
+      unit: "%"
+      name: "Poverty rate"
+    }
+    left_tiles {
+      title: "Schools per capita vs. Poverty rate"
+      type: SCATTER
+    }
+    right_tiles {
+      title: "Schools per capita vs. Poverty rate"
+      type: BIVARIATE
+    }
+  }
+
+  blocks {
+    title: "School amenities"
```

Comment:
Rather than amenities vs. poverty, I wonder if comparing drop-out rate is more interesting?

Two other comparison topics:
- mean monthly wages vs. poverty or illiteracy
- covid19 vs. poverty

[selected SVs](https://autopush.datacommons.org/tools/timeline#place=wikidataId%2FQ1445&statsVar=Mean_WagesMonthly_Worker__CumulativeCount_MedicalTest_ConditionCOVID_19_Positive__DropoutRate_Student_PrimarySchool__DropoutRate_Student_SecondarySchool__DropoutRate_Student_MiddleSchool&chart=%7B%22dropoutRate%22%3A%7B%22pc%22%3Afalse%2C%22delta%22%3Afalse%7D%2C%22cumulativeCount%22%3A%7B%22pc%22%3Afalse%2C%22delta%22%3Afalse%7D%7D)

Reply:
thanks, added those charts